apps/indexer/lib/indexer/token_balances.ex

```diff
@@ -51,6 +59,17 @@ defmodule Indexer.TokenBalances do
     Map.merge(token_balance, %{value: nil, value_fetched_at: nil, error: error_message})
   end
 
+  defp schedule_unfetched_token_balances(unfetched_token_balances) do
```

Comment:
This could also schedule other token balances, right?

```suggestion
  defp schedule_token_balances(token_balances) do
```

I think when you do:

```
    unfetched_token_balances(fetched_token_balances)
    |> schedule_token_balances
```

It is clear that you'll be scheduling the unfetched token balances.